From 929db4d86e67492bb3f345c2783c4983c9608bb0 Mon Sep 17 00:00:00 2001
From: Vineeth Raj <contact.twn@opmbx.org>
Date: Wed, 3 Sep 2014 22:59:53 +0530
Subject: [PATCH] Revert "Prebuilt chromium: Run a check for target device
 directory"

This reverts commit ce06d4cec2a9f5cb9a796f2c1bd705b87db0b6ff.
---
 envsetup.sh | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/envsetup.sh b/envsetup.sh
index f29981c..f4c9ef1 100644
--- a/envsetup.sh
+++ b/envsetup.sh
@@ -1357,19 +1357,13 @@ function chromium_prebuilt() {
     T=$(gettop)
     export TARGET_DEVICE=$(get_build_var TARGET_DEVICE)
     hash=$T/prebuilts/chromium/$TARGET_DEVICE/hash.txt
-    libsCheck=$T/prebuilts/chromium/$TARGET_DEVICE/lib/libwebviewchromium.so
-    frameworksCheck=$T/prebuilts/chromium/$TARGET_DEVICE/framework/webview/paks
-    device_target=$T/prebuilts/chromium/$TARGET_DEVICE/
 
-    if [ -r $hash ] && [ $(git --git-dir=$T/external/chromium/.git --work-tree=$T/external/chromium rev-parse --verify HEAD) == $(cat $hash) ] && [ -f $libsCheck ] && [ -d $frameworksCheck ]; then
+    if [ -r $hash ] && [ $(git --git-dir=$T/external/chromium/.git --work-tree=$T/external/chromium rev-parse --verify HEAD) == $(cat $hash) ]; then
         export PRODUCT_PREBUILT_WEBVIEWCHROMIUM=yes
         echo "** Prebuilt Chromium is up-to-date; Will be used for build **"
     else
         export PRODUCT_PREBUILT_WEBVIEWCHROMIUM=no
-        rm -rfv $device_target
-        echo ""
         echo "** Prebuilt Chromium out-of-date/not found; Will build from source **"
-        echo ""
     fi
 }
 
